lib/sentry/client.ex

```diff
@@ -21,7 +21,7 @@ defmodule Sentry.Client do
   """
   @spec send_event(%Event{}) :: {:ok, String.t} | :error
   def send_event(%Event{} = event) do
```

Comment:
Basically we should be injecting dependences in function signature so we can easily test and move private functions to its own module which we can also test independently so it could be like:

```elixir
def send_event(%Event{} = event, dsn_parser \\ DsnParser) do
{endpoint, public_key, secret_key} = dsn_parser.parse!
# ....
end
```

Unfortunately I don't ave time to implement it yet, but we can refactor it soon